tst/database/parse-server.spec.tsx

```diff
@@ -26,9 +30,90 @@ describe("ParseServer Tests", () => {
     expect(ParseMock.Object.extend).toHaveBeenCalledWith("Event");
   });
 
-  it("get events from DB", async () => {
+  it("create a new event", async () => {
+    const event = database.createEvent({
+      title: "My Protest Title",
+      description: "My protest description.",
+      location: "Los Angeles, CA",
+      imageSource: "https://www.pngrepo.com/png/103442/180/protest.png",
+      dateTime: new Date(2020, 8, 13),
+    });
+    expect(event).toBeInstanceOf(Promise);
+  });
+
+  it("get a specific event", async () => {
+    const event = database.getEvent("someObjectId");
+    expect(ParseMock.Query).toHaveBeenCalledTimes(1);
+    event.then(
+      (result: Parse.Object) => {
+        expect(result).not.toBeNull();
+      },
+      (_error: any) => {
+        fail();
+      }
+    );
```

Comment:
Need to add this to the "get list of events", "update an event", and "delete an event" tests. Could probably create a generic Promise check method. Will update in next revision.